Mathlib/Computability/AkraBazzi/GrowsPolynomially.lean

```diff
@@ -334,19 +404,34 @@ protected lemma add {f g : ℝ → ℝ} (hf : GrowsPolynomially f) (hg : GrowsPo
                           · exact (hf u hu).2
                           · exact (hg u hu).2
 
-protected lemma inv {f : ℝ → ℝ} (hf : GrowsPolynomially f) (hf' : ∀ᶠ x in atTop, 0 < f x) :
+protected lemma inv {f : ℝ → ℝ} (hf : GrowsPolynomially f) (hf' : ∀ᶠ x in atTop, f x ≠ 0) :
```

Comment:
You shouldn't need `hf'` here: either `f` is eventually positive, or it is eventually negative, or it is eventually zero. In the last case, `1/f` is also eventually zero, so the conclusion still holds.

Reply:
Also fixed.